In [7]:
import pandas as pd
from pathlib import Path
csv_path = Path.home() / "Downloads" /"Global Trends in Mental Health Disorder"/"Mental health Depression disorder Data.csv"   # <-- change this
df_mental_health = pd.read_csv(csv_path)
df_mental_health.head(30)


/var/folders/zs/lhn3v0dx0v32gt7hm_r0m1zc0000gn/T/ipykernel_63566/991666583.py:4: DtypeWarning: Columns (5,6) have mixed types. Specify dtype option on import or set low_memory=False.
  df_mental_health = pd.read_csv(csv_path)


index       Entity Code  Year Schizophrenia (%) Bipolar disorder (%)  \
0       0  Afghanistan  AFG  1990           0.16056             0.697779   
1       1  Afghanistan  AFG  1991          0.160312             0.697961   
2       2  Afghanistan  AFG  1992          0.160135             0.698107   
3       3  Afghanistan  AFG  1993          0.160037             0.698257   
4       4  Afghanistan  AFG  1994          0.160022             0.698469   
5       5  Afghanistan  AFG  1995          0.160076             0.698695   
6       6  Afghanistan  AFG  1996          0.160249             0.698914   
7       7  Afghanistan  AFG  1997          0.160554             0.699127   
8       8  Afghanistan  AFG  1998          0.160931             0.699372   
9       9  Afghanistan  AFG  1999          0.161311             0.699674   
10     10  Afghanistan  AFG  2000          0.161621             0.700015   
11     11  Afghanistan  AFG  2001          0.161957             0.700499   
12     12  Afghanistan  AFG  2002          0.162414             0.701141   
13     13  Afghanistan  AFG  2003          0.162916              0.70186   
14     14  Afghanistan  AFG  2004          0.163377             0.702556   
15     15  Afghanistan  AFG  2005          0.163706             0.703078   
16     16  Afghanistan  AFG  2006          0.163977             0.703517   
17     17  Afghanistan  AFG  2007          0.164302             0.703998   
18     18  Afghanistan  AFG  2008          0.164639              0.70448   
19     19  Afghanistan  AFG  2009          0.164932             0.704925   
20     20  Afghanistan  AFG  2010           0.16513             0.705313   
21     21  Afghanistan  AFG  2011          0.165272             0.705688   
22     22  Afghanistan  AFG  2012          0.165424             0.706086   
23     23  Afghanistan  AFG  2013          0.165579             0.706471   
24     24  Afghanistan  AFG  2014          0.165742             0.706876   
25     25  Afghanistan  AFG  2015          0.165895             0.707275   
26     26  Afghanistan  AFG  2016          0.166035             0.707686   
27     27  Afghanistan  AFG  2017          0.166158             0.708089   
28     28      Albania  ALB  1990          0.193413             0.695013   
29     29      Albania  ALB  1991          0.193393             0.695294   

   Eating disorders (%)  Anxiety disorders (%)  Drug use disorders (%)  \
0              0.101855               4.828830                1.677082   
1              0.099313               4.829740                1.684746   
2              0.096692               4.831108                1.694334   
3              0.094336               4.830864                1.705320   
4              0.092439               4.829423                1.716069   
5               0.09098               4.828337                1.728112   
6              0.089709               4.828083                1.737643   
7              0.088372               4.827726                1.746891   
8               0.08733               4.826971                1.756963   
9              0.086267               4.826413                1.770791   
10             0.086021               4.827047                1.788395   
11             0.086517               4.831409                1.839123   
12             0.087023               4.838318                1.934326   
13             0.087189               4.845538                2.051106   
14             0.088158               4.851512                2.163044   
15             0.088933               4.854684                2.247443   
16             0.090054               4.856685                2.321020   
17             0.091688               4.858861                2.405564   
18             0.093589               4.860437                2.483862   
19             0.095166               4.861533                2.543884   
20             0.097327               4.862777                2.571349   
21             0.09

In [4]:
rows,cols = df_mental_health.shape
print("Rows:", rows, "Columns:", cols)

Rows: 108553 Columns: 11


In [5]:
df_mental_health.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 108553 entries, 0 to 108552
Data columns (total 11 columns):
 #   Column                     Non-Null Count   Dtype  
---  ------                     --------------   -----  
 0   index                      108553 non-null  int64  
 1   Entity                     108553 non-null  object 
 2   Code                       103141 non-null  object 
 3   Year                       108553 non-null  object 
 4   Schizophrenia (%)          25875 non-null   object 
 5   Bipolar disorder (%)       19406 non-null   object 
 6   Eating disorders (%)       100236 non-null  object 
 7   Anxiety disorders (%)      6468 non-null    float64
 8   Drug use disorders (%)     6468 non-null    float64
 9   Depression (%)             6468 non-null    float64
 10  Alcohol use disorders (%)  6468 non-null    float64
dtypes: float64(4), int64(1), object(6)
memory usage: 9.1+ MB


Dataset: "Global Trends in Mental Health Disorder"
1. This dataset contains 108553 rows and 11 columns.
2. Columns include country,year and  different types of mental health disorders
3. This dataset helps to analyze prevelence of mental health disorders in different countries over years.

In [6]:
df_mental_health.columns

Index(['index', 'Entity', 'Code', 'Year', 'Schizophrenia (%)',
       'Bipolar disorder (%)', 'Eating disorders (%)', 'Anxiety disorders (%)',
       'Drug use disorders (%)', 'Depression (%)',
       'Alcohol use disorders (%)'],
      dtype='object')

Columns: 
'index'                     - qualitative    - It’s just an identifier
'Entity'                    - qualitative    - Represented by differnt countries as a text category .
'Code'                      - qualitative    - Abbreviation of a country (stored as text)
'Year'                      - quantitative   - Represents numberic time values that can be ordered
'Schizophrenia (%)'         - quantitative   - represents % of population with Schizophrenia, a numeric measure
'Bipolar disorder (%)'      - quantitative   - represents % of population with Bipolar disorder a numeric measure
'Eating disorders (%)'      - quantitative   - represents % of population with Eating disorders, a numeric measure
'Anxiety disorders (%)'     - quantitative   - represents % of population with Anxiety disorders, a numeric measure
'Drug use disorders (%)'    - quantitative   - represents % of population with Drug use disorders, a numeric measure
'Depression (%)'            - quantitative   - represents % of population with Depression, a numeric measure
'Alcohol use disorders (%)' - quantitative   - represents % of population with Alcohol use disorders, a numeric measure